pkg/corerp/processors/extenders/processor.go

```diff
@@ -17,7 +17,7 @@ type Processor struct {
 // the recipe output with the existing values in the resource. It returns an error if the secret values are not of type string
 // or if any of the other validations fail.
 func (p *Processor) Process(ctx context.Context, resource *datamodel.Extender, options processors.Options) error {
-	validator := processors.NewValidator(&resource.ComputedValues, &resource.SecretValues, &resource.Properties.Status.OutputResources)
+	validator := processors.NewValidator(&resource.ComputedValues, &resource.SecretValues, &resource.Properties.Status.OutputResources, resource.Properties.Status.Recipe)
```

Comment:
Should this be passed by reference?

```suggestion
	validator := processors.NewValidator(&resource.ComputedValues, &resource.SecretValues, &resource.Properties.Status.OutputResources, &resource.Properties.Status.Recipe)
```

Reply:
As discussed offline, this is a pointer. Will keep as is